src/viam/sdk/components/generic/generic.hpp

```diff
@@ -5,6 +5,7 @@
 
 #include <google/protobuf/descriptor.h>
 
+#include <viam/sdk/common/exception.hpp>
```

Comment:
(q) why include this if it's never used in the header?

Reply:
No reason -- we should look into adding that tool that Andrew mentioned which ensures that headers match the code.